posthog/api/test/test_plugin.py

```diff
@@ -1409,6 +1443,10 @@ def test_plugin_config_list(self, mock_get, mock_reload):
                     "plugin_info": None,
                     "delivery_rate_24h": None,
                     "created_at": mock.ANY,
+                    "updated_at": mock.ANY,
+                    "name": "ui name",
+                    "description": "ui description",
+                    "deleted": True,
```

Comment:
Shouldn't we skip returning deleted ones? Just to avoid leaking our soft deletes

Reply:
I have this in a follow-up PR